hippunfold/workflow/rules/warps.smk

```diff
@@ -251,7 +251,7 @@ rule create_warps_dentate:
             dir="AP",
             label="dentate",
             suffix="coords.nii.gz",
-            desc="init",
+            desc="laplace",
```

Comment:
Currently, they're the same thing actually (see rule linked below)
However, if we do end up refining the Laplace fields in the dentate in the future, then this is the filename we will want to use. 

https://github.com/khanlab/hippunfold/blob/80d35b719ee63e5bbb79bfdadec79432083214c5/hippunfold/workflow/rules/autotop.smk#L86

Reply:
ah ok, I'm guessing we added that cp rule to avoid changing later rules.. I would prefer actually removing that cp rule to avoid confusion, but can perhaps can clean that up in another PR..